docs/inventory-plugins.rst

```diff
@@ -1,229 +1,117 @@
 .. _ug_inventories_plugins:
 
 Inventory Plugins
-===================
-
-.. index::
```

Comment:
(same)

Reply:
Thanks for the suggestions. Since tests are passing, splitting this out now will require more unnecessary work. If we need to make further changes after this PR is merged, we can open another PR to include those changes.